micrometer-core/src/main/java/io/micrometer/core/instrument/MeterRegistry.java

```diff
@@ -263,6 +263,18 @@ public Search value(Statistic statistic, double value) {
             return this;
         }
 
+        public Optional<Double> firstValue(){
```

Comment:
I know you've tried to create this before :). My objection to it is it is only useful for counters and gauges that have a single measurement.